### PR TITLE
The initial requirements.

### PR DESCRIPTION
**Pull request by JesperMell**
This is the initial requirements for the CanBus node.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
+* [ReqID:003] It should be possible to unpack the signals.
+* [ReqID:004] It should be possible to set CAN in loopback mode.
+* [ReqID:005] It should be possible to send messages to the bus <write_interval>.
+* [ReqID:006] It should be possible to read messages from the bus <read_interval>.
+* [ReqID:007] It should be possible to generate modules to specific node through a Python script.
+* [ReqID:008] It should be possible to have a json file containing all data for functions and nodes.
+* [ReqID:009] It should be possbile to generate module for DEV and PROD env.
+* [ReqID:010] It should be possbile to copy FlaxCAN to DEV env for developers.
+* [ReqID:011] It shall be possible to generate modules to each nodes accodring to the nodes needs/values.
+* [ReqID:012] It shall be possible to generate constants to CAN bus through Python.
```
Comment (farochmehri) [inline README.md:16]: Generate global constants for THE NODES through python
It means that you need to create a header file (common.h) which contains the global constants required by the nodes according to thier needs.

**Review by farochmehri — CHANGES_REQUESTED** [review]
Fix requirement 012.

File: README.md (hunk 1 above)
Comment (JesperMell) [reply README.md:16]: Thanks for the explanation, I will adjust this!

Comment (JesperMell) [reply README.md]: This has now been solved.

**Pull request merged**